src/tokens/number/z-index.json

```diff
@@ -0,0 +1,10 @@
+{
+  "number": {
+    "z_index": {
+      "alert": {
+        "value": "999",
```

Comment:
Does this need to be a string, would an integer not work?

Reply:
Oh, good point! 👍🏽

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/1928/commits/5ac2cfc8b3fb10596eeb8a57c8b72806aa7f227a